crates/wit-parser/src/lib.rs

```diff
@@ -112,14 +121,21 @@ pub enum AstItem {
 ///
 /// This is directly encoded as an "ID" in the binary component representation
 /// with an interfaced tacked on as well.
-#[derive(Debug, Clone, Hash, Eq, PartialEq, Ord, PartialOrd)]
+#[derive(Debug, Clone, Hash, Eq, PartialEq, Ord, PartialOrd, Serialize)]
+#[serde(into = "String")]
 pub struct PackageName {
     /// A namespace such as `wasi` in `wasi:foo/bar`
     pub namespace: String,
     /// The kebab-name of this package, which is always specified.
     pub name: String,
     /// Optional major/minor version information.
-    pub version: Option<Version>,
+    pub version: Option<SerializableVersion>,
```

Comment:
Instead of using `SerializableVersion` could this use `serialize_with`?

Reply:
According to the tests, this works with `semver::Version` without `SerializableVersion` at all. So I removed it in be17ced3d194c72c22c6b158e66dca52ad0d8e4e.